### PR TITLE
Feature/#75

### PR DESCRIPTION
**Pull request by colefoster**
Added dynamic options to past attendance dropdown per existing attendance

### CODE REVIEW

File: frontend/scripts/index.js [hunk 1]
```diff
@@ -45,8 +47,24 @@ function addAttendance(attendance_ID) {
...
     xmlHttp.send(JSON.stringify(attendance_json));
     alert(xmlHttp.responseText);
+    return xmlHttp;
 }
 
+function fillAttendanceDropdown() {
+    const dropDown = document.getElementById("select-5c86");
+    const options = dropDown.getElementsByTagName("option");
+    const attendance_json = JSON.parse(getSummary().responseText);
+    console.log(attendance_json);
```
Comment (ehharvey) [inline frontend/scripts/index.js:57]: Remove these `console.log`s

File: frontend/scripts/index.js [hunk 2]
```diff
@@ -45,8 +47,24 @@ function addAttendance(attendance_ID) {
...
+    for (let i = 0; i < attendance_json.ids.length; i++) {
+        const newOption = document.createElement("option");
+        newOption.innerText = "Attendance " + attendance_json.ids[i];
+        dropDown.appendChild(newOption);
+    }
+    //options[0].innerText = "Attendance " + attendance_json.ids[0];
```
Comment (ehharvey) [inline frontend/scripts/index.js:63]: Remove the dead code

File: frontend/scripts/index.js [hunk 3]
```diff
@@ -45,8 +47,24 @@ function addAttendance(attendance_ID) {
     xmlHttp.setRequestHeader("Content-Type", "application/json");
     xmlHttp.send(JSON.stringify(attendance_json));
     alert(xmlHttp.responseText);
+    return xmlHttp;
```
Comment (ehharvey) [inline frontend/scripts/index.js:50]: Why are we returning the http object? If we need to return anything, ideally we should just return the body (JSON dictionary)

File: frontend/scripts/index.js (hunk 1 above)
Comment (ehharvey) [reply frontend/scripts/index.js:57]: We can use the `logConsole` function

**Review by Justin200104 — APPROVED** [review]
Looks good to me

**Comment by colefoster** [comment]
Front end is 90% complete and working, implements viewing old attendances, and creating new ones successfully. (reload page to view recently created attendances in past attendances <- will fix next sprint)

**Review by ehharvey — APPROVED** [review]

**Pull request merged**